src/app/pagination/pagination.component.view.spec.ts

```diff
@@ -153,7 +155,7 @@ describe('MzPaginationComponent:view', () => {
         paginationPageButtons()[3].querySelector('a').click();
```

Comment:
Still wrong, for it to goes through `lastPage()` method you need to click on element at index 4 ....
[0] - First Page
[1] - Previous page
[2] - 1
[3] - 2
[4] - Next page
[5] - Last page